### DISCUSSION TITLE
Bug in Zelda Ocarina of Time

### DISCUSSION THREAD

**Discussion by TheLegendOfLeleo** [mupen64plus-ae/mupen64plus-ae#1049] [Q&A]
my problem started yesterday (as far as i remember), first i'm brazilian so if i write something wrong i'm sorry i was playing zelda like i always do, i was going to get the "lenses of truth" and that's when the problem started : my game crashed and just exited by itself and when I entered I realized that my progress had been lost and I had to go back from the last save point, but it wasn't something all at once, so far it's like this I'm in the shadow temple and it seems that the problem has gotten worse, it keeps crashing and restarting by itself, I've closed the app tab, restarted the phone and the problem continues

**Comment by fzurita** [comment]
Are you in the latest version of the app? If you are, you can probably fix this by selecting the game, home to settings, and changing the emulation profile to GLideN64-Medium.

**Reply by TheLegendOfLeleo** [reply]
My version is 3.0.322

**Reply by TheLegendOfLeleo** [reply]
my problem is solved, thank you very much
